core/views/files.py

```diff
@@ -320,15 +320,15 @@ def get(self, request, *args, **kwargs):
 
         if "clipboard" not in request.session.keys():
             request.session["clipboard"] = []
-        return super(FileView, self).get(request, *args, **kwargs)
+        return super().get(request, *args, **kwargs)
 
     def post(self, request, *args, **kwargs):
         self.object = self.get_object()
         if "clipboard" not in request.session.keys():
             request.session["clipboard"] = []
         if request.user.can_edit(self.object):
             # XXX this call can fail!
-            FileView.handle_clipboard(request, self.object)
+            self.handle_clipboard(request, self.object)
```

Comment:
```suggestion
            self.__class__.handle_clipboard(request, self.object)
```
Ce serait plus proche de ce qui se passe vraiment
J'ai pas vérifié le code qui est appelé mais selon comment c'est implémenté ce que t'appelle, tu peux ne pas avoir le même résultat si tu appelle la méthode de l'instance ou de la class elle même

Reply:
https://docs.python.org/3/library/functions.html#staticmethod

> Changed in version 3.10: Static methods now inherit the method attributes (\_\_module\_\_, \_\_name\_\_, \_\_qualname\_\_, \_\_doc\_\_ and \_\_annotations\_\_), have a new \_\_wrapped\_\_ attribute, and are now __callable as regular functions__.